pyam/logger.py

```diff
@@ -12,3 +13,12 @@ def logger():
         _LOGGER = logging.getLogger()
         _LOGGER.setLevel('INFO')
     return _LOGGER
+
+
+@contextmanager
+def adjust_log_level(level='ERROR'):
```

Comment:
@danielhuppmann what would happen if `adjust_log_level` is called before `logger`? Is that valid use case? Currently tests in #262 are failing because of that